packages/server/graphql/public/typeDefs/Query.graphql

```diff
@@ -52,6 +52,11 @@ type Query {
     isInvited: Boolean
   ): String
 
+  """
+  Get an OAuth 2.0 Application by ID
+  """
+  oauthAPIProvider(id: ID!): OAuthAPIProvider
```

Comment:
-1 why does this live on `Query` but `oauthApplications` live on `Organization`?

Since this requires being logged in & having access to the Org, it should live on Organization.